client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -432,6 +434,41 @@ class WpcomChecklistComponent extends PureComponent {
 		);
 	};
 
+	renderDomainVerifiedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate, siteSlug } = this.props;
+
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				bannerImageSrc="/calypso/images/stats/tasks/create-tagline.svg"
+				buttonText={ translate( 'Verify' ) }
+				completedButtonText={ translate( 'Change' ) }
+				completedTitle={ translate( 'You verified your domain name' ) }
+				description={ translate(
+					'Email assigned to your domains has to be verified so we can get in touch with you when needed.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Your email has to be verified so we can get in touch with you when needed.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-verification-nudge-to-checklist)

Reply:
We may need to be slightly stronger with the copy here -- we are _legally required_ to confirm the email address to keep the domain available. Without email verification, the domain simply won't work and won't allow the site to load. How about the following?
* `completedTitle` => `You verified the email address for your domain(s)`
* `description` => `The email address for your domains has to be verified to allow your domains to work properly.`

Might it also be worth having `Verify your domains`/`Verify %(domainName)s` be slightly more explicit? e.g. `Verify email address for domains`/`Verify email address for %(domainName)s`. This does confuse _our_ email confirmation, but it is separate/distinct.